devicehive-shim-kafka-impl/src/main/java/com/devicehive/shim/config/KafkaRpcConfig.java

```diff
@@ -86,4 +110,27 @@ public Properties serverConsumerProps() {
         props.put(ConsumerConfig.GROUP_ID_CONFIG,  requestConsumerGroup);
         return props;
     }
+    
+    public Properties topicProps() {
+        Properties props = new Properties();
+        props.setProperty("retention.ms", retentionMs);
+        return props;
+    }
+
+    public void createTopic(String topic) {
```

Comment:
This class is not the best place for 'createTopic' method, can we move it out to the more appropriate place?